docs/en/stock-management/stock.settings.md

```diff
@@ -0,0 +1,66 @@
+&raquo; [Home](../index.md) / [Inventory Management](./index.md) / Stock Settings
+
+# Stock Settings
+
+There a number of settings that affect how the BHIMA software deals with stock
+management.  To check and update the settings, one goes to the "Stock
+Settings" page (under the "Stock" menu):
+
+![Stock Settings Menu](./images/stock-settings-menu.png)
+
+If you do not see the "Stock Settings" menu item, consult your system
+adminstrator.  You need to be given permission to access that page.
+
+- To add permission for a user to access the "Stock Settings' page: Under
+  `Administration > Role Management`, click on the action menu on the right
+  end of the desired user.  In the action menu, click on `Permission`.  Scroll
+  down to "Stock Settings" (at the end of the Stock items), then click the check
+  box and the [Save] button.  The Stock Settings page should now be accessible
+  for the user.
+
+The Stock Settings Page allows access to the following items:
+
+- **Number of months for calculating the average monthly consumption** (integer)
```

Comment:
This determines the number of months in the past to use to calculate the average monthly consumption.   The AMC is a "moving" average, since it uses a window of only the previous X months to determine the average.  This setting will set the X.

Reply:
Updated